src/Options/Generic.hs

```diff
@@ -959,15 +977,18 @@ getWithHelp
     -> io (a, io ())
     -- ^ (options, io action to print help message)
 getWithHelp desc = do
-  a <- liftIO (Options.customExecParser defaultParserPrefs info)
+  a <- getRecord' header mempty
   return (a, help)
   where
     header = Options.header (Data.Text.unpack desc)
-    info = Options.info parseRecord header
-    help = liftIO (showHelpText defaultParserPrefs info)
+    info   = Options.info parseRecord header
+    help   = liftIO (showHelpText (Options.prefs defaultParserPrefs) info)
```

Comment:
No because `header` is a modifier for a `ParserInfo`, not a `ParserPrefs` but it is given to `info`.